### PR TITLE
[TESTED] Show deleted characters differently to non deleted ones

### PR DESCRIPTION
**Pull request by milan60**
Deleted characters now show up differently than non deleted ones. You can also hide deleted characters now.

### CODE REVIEW

**Comment by ExpDev07** [comment]
Wow you have really done some work! Going to pull this and check it out 😍

File: resources/js/Components/Card.vue [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 <script>
 export default {
+    props: ['deleted'],
```
Comment (ExpDev07) [inline resources/js/Components/Card.vue:24]: Put under `name` as 

```
name: 'Card',
props: {
  deleted: Boolean,
}
```

File: tailwind.config.js [hunk 2]
```diff
@@ -34,11 +34,27 @@ module.exports = {
...
+                'dark-warning-pale':   defaultTheme.colors.yellow['700'],
+                'dark-success-pale':   defaultTheme.colors.green['700'],
             }
         },
     },
     plugins: [
         require('@tailwindcss/typography'),
         require('@tailwindcss/forms'),
     ],
+    darkMode: 'class',
```
Comment (ExpDev07) [inline tailwind.config.js:59]: Put this on top under` mode: 'jit'`.

**Review by ExpDev07 — CHANGES_REQUESTED** [review]
Added some comments. I'll merge after these have been fixed. Otherwise, great job!

**Comment by milan60** [comment]
Implemented your fixes, i've also completed localization of the entire admin panel (english is the only language as of now) but i put that in a different branch so we can get these changes pushed through first

**Review by ExpDev07 — APPROVED** [review]
LGTM!

**Pull request merged**